src/platform/ios.rs

```diff
@@ -74,6 +75,16 @@ pub trait WindowExtIOS {
     /// and then calls
     /// [`-[UIViewController setNeedsStatusBarAppearanceUpdate]`](https://developer.apple.com/documentation/uikit/uiviewcontroller/1621354-setneedsstatusbarappearanceupdat?language=objc).
     fn set_prefers_status_bar_hidden(&self, hidden: bool);
+
+    /// Sets whether the [`Window`] prefers the status bar hidden.
+    ///
+    /// The default is system-defined.
+    ///
+    /// This changes the value returned by
+    /// [`-[UIViewController preferredStatusBarStyle]`](https://developer.apple.com/documentation/uikit/uiviewcontroller/1621416-preferredstatusbarstyle?language=objc),
+    /// and then calls
+    /// [`-[UIViewController setNeedsStatusBarAppearanceUpdate]`](https://developer.apple.com/documentation/uikit/uiviewcontroller/1621354-setneedsstatusbarappearanceupdat?language=objc).
+    fn set_preferred_status_bar_style(&self, status_bar_style: UIStatusBarStyle);
```

Comment:
Since `UIStatusBarStyle` is private, this will not actually be usable.

Create a new public enum in this file instead.